.gitignore

```diff
@@ -63,3 +63,5 @@ crytic-export
 *.vsix
 
 bin
+op-monitorism/.air.toml
```

Comment:
```suggestion
```

Reply:
Thanks forgot to remove this one from `op-monitorism` will keep this one for `op-defender`.